### ISSUE TITLE
What should I do for block builder to send a block to relay/relays?

### ISSUE THREAD

**Issue by yanikitat** [flashbots/builder#75]
Hello guys! I'm investigating your projects a couple days and I have some problems/questions.

1. Why when I call `eth_sendBundle` rpc call of my flashbots block builder I'm getting empty response? Or, if to be more precise, I'm getting an empty BundleHash. This is how response looks like:
`{BundleHash:}`
But! When I send exactly the same request to the public `https://boost-relay-goerli.flashbots.net` I'm getting the `BundleHash`
2. I have some trouble with documentation of builder in general. I can't find the list of the supported rpc call (I have found [this](https://docs.flashbots.net/flashbots-auction/searchers/advanced/rpc-endpoint), but i'm not sure that this is about block builder).
3. And the main point that I don't understand -- what should I do for block builder to send a block to relay/relays? Maybe even empty block or block with one tx. Now I just want to test it, make sure that my block builder is working and look how it works.

This is how I run my block builder:
`./build/bin/geth --goerli --builder --builder.remote_relay_endpoint https://boost-relay-goerli.flashbots.net --log.debug`
I'm also add `BUILDER_TX_SIGNING_KEY` env var

I will glad if somebody will be able to help me and explain how it works:)
Also if you need more info please let me know

**Comment by yanikitat** [comment]
Okay, I have ran prysm consensus client, but I can't understand does block builder gets trigger from prysm for block building or not. Can block builder build block while syncing?

**Comment by yanikitat** [comment]
No, but I have figured out it by myself. I can write my solution and close this issue if necessary

**Comment by aaroua5** [comment]
Yes Please

**Comment by aaroua5** [comment]
@yanikitat  can you explain please ?

**Comment by manishparanjape** [comment]
@yanikitat  can you provide details here please?

**Comment by ezynda3** [comment]
I've modified this on my own personal instance and can make a PR if that's something people want.

**Comment by dewindtk** [comment]
@yanikitat @ezynda3 I am actually struggling with the same thing, I am discussing my issue on the flashbots discord without yet success: https://discord.com/channels/755466764501909692/1172625715632549980

Basically, I eth_sendBundle a transaction through the geth console, this bundle gets recognized, the payload gets updated sucessfully but is never sent to the Relay. Operating on my private devnet.

I would immensely appreciate ANY help on this matter

**Comment by manishparanjape** [comment]
@ezynda3 Can you share your PR please?

**Comment by forencen** [comment]
hello, I'm having the same problem, I can't get the BundleHash, what should I if solve it?
